microk8s-resources/actions/enable.falco.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+
+microk8s enable dns
+
+NAMESPACE_PTR="sysdig-agent"
+
+MANIFEST_PTR="https://raw.githubusercontent.com/draios/sysdig-cloud-scripts/master/agent_deploy/kubernetes/sysdig-agent-service.yaml"
```

Comment:
I think getting the manifest from the master will be a problem.
- Are you certain the master will always work? It is not a release so there might be a commit that temporarily breaks the master causing microk8s.enable falco to fail.
- It makes it really hard for us to have reproducible deployments. Let's say someone opens a issue saying 'falco did not work for me', how would you know what did the user deploy?
- You pair a manifest with the default values that lives in the snap with the manifests in the master branch. How can you be sure that the default values manifest will stay exactly the same as falco evolves?
- Are you sure that the falco master branch will deploy correctly on kubernetes v1.21 after 2 year? I am not.

Please select a specific release to deploy from https://github.com/falcosecurity/falco/releases